KsApi/models/graphql/adapters/Project.Country+CountryFragment.swift

```diff
@@ -1,10 +1,26 @@
-import Foundation
+import Prelude
 
 extension Project.Country {
-  static func country(from countryFragment: GraphAPI.CountryFragment) -> Project.Country? {
-    guard let country = Project.Country.all
-      .first(where: { $0.countryCode == countryFragment.code.rawValue }) else { return nil }
+  static func country(from countryFragment: GraphAPI.CountryFragment,
+                      minPledge: Int,
+                      maxPledge: Int,
+                      currency: GraphAPI.CurrencyCode) -> Project.Country? {
+    guard let countryWithCurrencyDefault = Project.Country.all
+      .first(where: { $0.countryCode == countryFragment.code.rawValue }) else {
```

Comment:
Could use trailing closure syntax here and below but definitely not a big deal.

Reply:
So oddly this works:
```
    let state = Project.Country.all.first { $0.countryCode == countryFragment.code.rawValue }
    
    guard let countryWithCurrencyDefault = state else {
      return nil
    }
```
but this does not:
```
    guard let countryWithCurrencyDefault = Project.Country.all.first { $0.countryCode == countryFragment.code.rawValue } else {
      return nil
    }
```

Getting a bunch of errors here, so I think I'll leave it for now, as the former doesn't really save us much space/is necessarily cleaner.